### PR TITLE
fix(sdk): automatically create std/ETH bridges

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Updates the SDK to automatically create the Standard and ETH bridges as long as the L1StandardBridge provider is supplied.

### CODE REVIEW

File: packages/sdk/src/utils/contracts.ts [hunk 1]
```diff
@@ -165,19 +165,19 @@ export const getBridgeAdapters = (
...
 ): BridgeAdapters => {
   const adapterData: BridgeAdapterData = {
-    ...(CONTRACT_ADDRESSES[l2ChainId]
+    ...(CONTRACT_ADDRESSES[l2ChainId] || opts?.contracts?.l1?.L1StandardBridge
       ? {
           Standard: {
             Adapter: StandardBridgeAdapter,
             l1Bridge:
-              opts.contracts?.l1?.L1StandardBridge ||
+              opts?.contracts?.l1?.L1StandardBridge ||
```
Comment (roninjin10) [inline packages/sdk/src/utils/contracts.ts:173]:
```suggestion
              opts?.contracts?.l1?.L1StandardBridge ??
```
nit.  not at all important in this case just one of those random things I can't help but nit about

File: packages/sdk/src/utils/contracts.ts [hunk 2]
```diff
@@ -165,19 +165,19 @@ export const getBridgeAdapters = (
...
               CONTRACT_ADDRESSES[l2ChainId].l1.L1StandardBridge,
             l2Bridge: predeploys.L2StandardBridge,
           },
           ETH: {
             Adapter: ETHBridgeAdapter,
             l1Bridge:
-              opts.contracts?.l1?.L1StandardBridge ||
+              opts?.contracts?.l1?.L1StandardBridge ||
```
Comment (roninjin10) [inline packages/sdk/src/utils/contracts.ts:180]:
```suggestion
              opts?.contracts?.l1?.L1StandardBridge ??
```
nit

**Review by roninjin10 — APPROVED** [review]

**Comment by tynes** [comment]
This code is bouncy

**Review by tynes — APPROVED** [review]

**Review by tynes — CHANGES_REQUESTED** [review]
Actually, this looks like its breaking something now

```
  1) Basic L1<>L2 Communication
       "before all" hook in "Basic L1<>L2 Communication":
     Error: Invalid address
      at assert (/opt/optimism/packages/sdk/src/utils/assert.ts:3:11)
      at toAddress (/opt/optimism/packages/sdk/src/utils/coercion.ts:106:11)
      at new StandardBridgeAdapter (/opt/optimism/packages/sdk/src/adapters/standard-bridge.ts:52:16)
      at getBridgeAdapters (/opt/optimism/packages/sdk/src/utils/contracts.ts:192:28)
      at new CrossChainMessenger (/opt/optimism/packages/sdk/src/cross-chain-messenger.ts:176:37)
```

**Comment by smartcontracts** [comment]
> Actually, this looks like its breaking something now
> 
> ```
>   1) Basic L1<>L2 Communication
>        "before all" hook in "Basic L1<>L2 Communication":
>      Error: Invalid address
>       at assert (/opt/optimism/packages/sdk/src/utils/assert.ts:3:11)
>       at toAddress (/opt/optimism/packages/sdk/src/utils/coercion.ts:106:11)
>       at new StandardBridgeAdapter (/opt/optimism/packages/sdk/src/adapters/standard-bridge.ts:52:16)
>       at getBridgeAdapters (/opt/optimism/packages/sdk/src/utils/contracts.ts:192:28)
>       at new CrossChainMessenger (/opt/optimism/packages/sdk/src/cross-chain-messenger.ts:176:37)
> ```

Fixed ser

**Pull request merged**